public/index.html

```diff
@@ -35,15 +35,37 @@
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
     <title>Randomizator</title>
     <style>
-      body{
+      * {
+        scrollbar-color: #771e76 #1b1b1b;
+        scrollbar-width: thin;
+      }
+
+      *::-webkit-scrollbar-track {
+        background-color: #1b1b1b;
+      }
+
+      *::-webkit-scrollbar {
+        height: 4px;
+        width: 8px;
+        background-color: #1b1b1b;
+      }
+
+      *::-webkit-scrollbar-thumb {
+        background-color: #771e76;
+        border-radius: 10px;
+      }
+
+      body {
         background: #1B1B1B;
         margin:0;
         position: relative;
         min-height: 100vh;
       }
-      input{
+
+      input {
         transition: all 0.6s linear;
       }
```

Comment:
Jeśli klasa ma tylko jedną własność to można ją wtedy zapisać w jednej linii, wtedy ta własność nie będzie się czuła samotnie :smiley_cat: